php/Dockerfile

```diff
@@ -1,6 +1,7 @@
 ARG WASI_SDK_VERSION
 FROM ghcr.io/vmware-labs/wasmlabs/wasi-builder:${WASI_SDK_VERSION}
 RUN DEBIAN_FRONTEND=noninteractive apt install -y \
+      libtool-bin \
       bison \
       re2c \
       libsqlite3-dev \
```

Comment:
I think this should be removed. But we would need to ensure CLFLAGS for PHP with SQLite points to or lib, and not to the headers installed into the system.

Reply:
It seems PHP's `./configure` doesn't provide a `--with-sqlite3[=DIR]` knob or similar.
So, I guess we would need to keep using system libraries.

If really needed, we could set some symlinks in the container to point to our libraries, but I don't see the need for that right now.